src/main/java/net/mcreator/ui/workspace/selector/WorkspaceSelector.java

```diff
@@ -408,7 +408,8 @@ private void addWorkspaceButton(String text, ImageIcon icon, ActionListener even
 
 	private void initWebsitePanel() {
 		CompletableFuture<String[]> newsFuture = new CompletableFuture<>();
-		MCreatorApplication.WEB_API.getWebsiteNews(newsFuture);
+		if (MCreatorApplication.isInternet)

```

Comment:
If you want to show your connection error message, you should omit the if here so data is still loaded, same for motw below

Reply:
✔️ Fixed